core/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.36.1 - [unreleased]
```

Comment:
Oh, missed this one. @futpib can you please:

1. Update the version in `core/Cargo.toml` to `0.36.1`
2. Update `libp2p-core` to `0.36.1` in the root `Cargo.toml`
3. And reference the bump in the root `Changelog`.

See also https://github.com/libp2p/rust-libp2p/blob/master/docs/release.md#development-between-releases